mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule ClusterConsul.MixProject do
   def project do
     [
       app: :libcluster_consul,
-      version: "1.1.1",
+      version: "1.1.2",
```

Comment:
🎨 According to [SemVer](https://semver.org/) this should be `1.2.0` given you are adding functionality in a backwards-compatible way

Reply:
Good point, thanks.